core/domain/customization_args_util_test.py

```diff
@@ -390,48 +399,41 @@ def test_frontend_customization_args_defs_coverage(self):
                     line
                 ))
             if ca_frontend_interface_match:
-                actual_interaction_ca_frontend_interfaces.add(
+                interaction_ids_with_ca_frontend_interfaces.add(
                     ca_frontend_interface_match.group(2))
 
-        expected_interaction_ca_interfaces = (
-            set(interaction_registry.Registry.get_all_interaction_ids()))
-
+        self.assertTrue(len(interaction_ids_with_ca_backend_interfaces) > 0)
         self.assertEqual(
-            expected_interaction_ca_interfaces,
-            actual_interaction_ca_backend_interfaces)
+            all_interaction_ids,
+            interaction_ids_with_ca_backend_interfaces)
+
+        self.assertTrue(len(interaction_ids_with_ca_frontend_interfaces) > 0)
         self.assertEqual(
-            expected_interaction_ca_interfaces,
-            actual_interaction_ca_frontend_interfaces)
+            all_interaction_ids,
+            interaction_ids_with_ca_frontend_interfaces)
 
     def test_frontend_customization_args_constructor_coverage(self):
         """Test to ensure that InteractionObjectFactory.ts covers constructing
         customization arguments for each interaction. Uses regex to confirm
         that the CustomizationArgs interface is imported for each interaction
```

Comment:
In next PR: you've removed this check so perhaps update docstring accordingly

Reply:
Done